### PR TITLE
dom0_cfg: Configure image and dtb for the Domain config

### PR DESCRIPTION
**Pull request by oleksiimoisieiev**
Add domd image and dtb sections and load_addr_offset for the domain
config.
Where:
   img_start - set to the beggining of the section where domain binary
   was placed
   img_end - end of the section where domain binary was placed
   dtb_start - start of the section where domain device tree was
   placed
   dtb_end - end of the section where domain device tree was placed

### CODE REVIEW

File: src/domd_cfg.c [hunk 1]
```diff
@@ -374,6 +374,13 @@ static uint32_t domd_irqs[] = {
 			185,
 };
 
+extern char __img_domd_start[];
+extern char __img_domd_end[];
+extern char __dtb_domd_start[];
+extern char __dtb_domd_end[];
+
+#define LOAD_ADDR_OFFSET 0x80000
```
Comment (lorc) [inline src/domd_cfg.c]: Commit description does not mention this. What is this offset and why it is equals to 0x80000?

**Comment by sa-kib** [comment]
LGTM

**Comment by firscity** [comment]
`Reviewed-by: Dmytro Firsov <dmytro_firsov@epam.com>`

**Comment by oleksiimoisieiev** [comment]
Added RB @firscity

**Pull request merged**